### PR TITLE
Update the README for mainnet

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,19 +7,27 @@ More information about the protocol in this [presentation](https://youtu.be/Jj8r
...
    We are running an `asb` instance on testnet.
    You can swap with to get familiar with the `swap` CLI.
    Our peer id is `12D3KooWCdMKjesXMJz1SiZ7HgotrxuqhQJbP5sgBm2BwP1cqThi` and our multiaddress is `/dnsaddr/xmr-btc-asb.coblox.tech`
 3. Follow the instructions printed to the terminal
 
-## Limitations
-
-For now, the MVP is limited to `testnet3` on Bitcoin and `stagenet` on Monero.
+For running the software on mainnet you just omit the `--testnet` flag.
+Running on mainnet will automatically apply sane defaults for mainnet.
```
Comment (bonomat) [inline README.md:25]: A bit repetitive.
```suggestion
Running on mainnet will automatically apply sane defaults.
```

**Review by bonomat — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -7,19 +7,27 @@ More information about the protocol in this [presentation](https://youtu.be/Jj8r
 
 ## Quick start - CLI
 
+From version `0.6.0` onwards the software default to running on `mainnet`.
+It is recommended to try the software on testnet first, which can be achieved by providing the `--testnet` flag.
+This quickstart guide assumes that you are running the software on (i.e. Bitcoin testnet3 and Monero stagenet):
```
Comment (thomaseizinger) [inline README.md:12]: Is there another `testnet` missing after the `on`?

File: README.md [hunk 3]
```diff
@@ -7,19 +7,27 @@ More information about the protocol in this [presentation](https://youtu.be/Jj8r
...
-## Limitations
-
-For now, the MVP is limited to `testnet3` on Bitcoin and `stagenet` on Monero.
+For running the software on mainnet you just omit the `--testnet` flag.
+Running on mainnet will automatically apply sane defaults for mainnet.
+Be aware that this software is still early-stage.
+Make sure to check `--help` and understand how the `cancel` and `refund` commands work before running on mainnet.
+You are running this software at your own risk.
+As always we recommend: Verify, don't trust.
+All code is available in this repo.
```
Comment (thomaseizinger) [inline README.md:30]:
```suggestion
All code is available in this repository.
```

**Review by thomaseizinger — APPROVED** [review]
concept ACK

Some minor suggestions.

**Pull request merged**